cli/src/funTest/assets/semver4j-analyzer-result.yml

```diff
@@ -13,8 +13,8 @@ repository:
   config:
     excludes:
       scopes:
-      - pattern: "testClasspath"
-        reason: "TEST_TOOL_OF"
+        - pattern: "testClasspath"
```

Comment:
I'm not sure whether we should align in *that* direction: While I'm aware that many YAML examples do have the leading spaces for [block sequence](https://yaml.org/spec/1.2.2/#821-block-sequences) entries, our serialized files actually seem not to use them...

Reply:
BTW, "consistent" to what? Other block sequences in the same file, or to other files?

Reply:
> I'm not sure whether we should align in that direction: While I'm aware that many YAML examples do have the leading spaces for [block sequence](https://yaml.org/spec/1.2.2/#821-block-sequences) entries, our serialized files actually seem not to use them...

As most part's of the file have been using that identation before, I believe it's ok to fix the few inconsistencies.
I agree that it should be consistent also with other files, but in scope of this PR I believe local consistency is ok.

> BTW, "consistent" to what? Other block sequences in the same file, or to other files?

local consistency.

Note: I was planning to make an upcoming change which aligns the formatting with other files in a separate PR.

Reply:
If you plan to align with the "correct" formatting anyway, why do this commit in the first place? I don't see the benefit over just fixing it right away.

Reply:
The otherway around it is way more effort, and now the problem is already fixed locally. I don't see the benefit of discussing this any further TBH, it is making the formatting consistent within the file. So, what's your aim in continuing this discussion?